--- a/Sources/OrderedJSON/JSONValue+ExpressibleByLiteral.swift
+++ b/Sources/OrderedJSON/JSONValue+ExpressibleByLiteral.swift
@@ -19,6 +19,6 @@
 extension JSONValue: ExpressibleByDictionaryLiteral {
   public init(dictionaryLiteral elements: (String, JSONValue)...) {
-    var dictionary = OrderedDictionary<String, JSONValue>()
+    var dictionary = JSONObject()
     dictionary.reserveCapacity(elements.count)
     for (key, value) in elements {
       dictionary[key] = value
